### ISSUE TITLE
[SonicFrontiers] 

### ISSUE THREAD

**Issue by sharpemu6898** [thesupersonic16/HedgeModManager#632]
HMM Info:
```
    Version: 7.10-0-5019cae
    Args: 
    StartDir: D:\SteamLibrary\steamapps\common\SonicFrontiers
    Process Level: User
    Game: Sonic Frontiers
    GameInstall: Sonic Frontiers
    Update Status: Up To Date
```

Exception:
```
    Type: Exception
    Message: Error compiling codes
(7,13): error CS1010: Newline in constant
(7,138): error CS1026: ) expected
(7,138): error CS1026: ) expected
(7,139): error CS1513: } expected

    Source: HedgeModManager
    Function: System.Threading.Tasks.Task CompileCodes[TCollection](TCollection, System.IO.Stream, System.String[])
    StackTrace: 
       at HedgeModManager.CodeCompiler.CodeProvider.CompileCodes[TCollection](TCollection sources, Stream resultStream, String[] loadPaths) in D:\a\HedgeModManager\HedgeModManager\HedgeModManager\CodeCompiler\CodeProvider.cs:line 109
       at HedgeModManager.CodeCompiler.CodeProvider.CompileCodes[TCollection](TCollection sources, String assemblyPath, String[] loadsPaths) in D:\a\HedgeModManager\HedgeModManager\HedgeModManager\CodeCompiler\CodeProvider.cs:line 62
       at HedgeModManager.ModsDB.<SaveDB>d__27.MoveNext()
    --- End of stack trace from previous location where exception was thrown ---
       at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
       at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
       at HedgeModManager.MainWindow.<SaveModsDB>d__42.MoveNext() in D:\a\HedgeModManager\HedgeModManager\HedgeModManager\UI\MainWindow.xaml.cs:line 651
    --- End of stack trace from previous location where exception was thrown ---
       at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
       at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
       at HedgeModManager.MainWindow.<SaveConfig>d__54.MoveNext() in D:\a\HedgeModManager\HedgeModManager\HedgeModManager\UI\MainWindow.xaml.cs:line 982
    InnerException: 
```

HMM Info:
```
    Version: 7.10-0-5019cae
    Args: 
    StartDir: D:\SteamLibrary\steamapps\common\SonicFrontiers
    Process Level: User
    Game: Sonic Frontiers
    GameInstall: Sonic Frontiers
    Update Status: Up To Date
```

Exception:
```
    Type: Exception
    Message: Error compiling codes
(7,13): error CS1010: Newline in constant
(7,138): error CS1026: ) expected
(7,138): error CS1026: ) expected
(7,139): error CS1513: } expected

    Source: HedgeModManager
    Function: System.Threading.Tasks.Task CompileCodes[TCollection](TCollection, System.IO.Stream, System.String[])
    StackTrace: 
       at HedgeModManager.CodeCompiler.CodeProvider.CompileCodes[TCollection](TCollection sources, Stream resultStream, String[] loadPaths) in D:\a\HedgeModManager\HedgeModManager\HedgeModManager\CodeCompiler\CodeProvider.cs:line 109
       at HedgeModManager.CodeCompiler.CodeProvider.CompileCodes[TCollection](TCollection sources, String assemblyPath, String[] loadsPaths) in D:\a\HedgeModManager\HedgeModManager\HedgeModManager\CodeCompiler\CodeProvider.cs:line 62
       at HedgeModManager.ModsDB.<SaveDB>d__27.MoveNext()
    --- End of stack trace from previous location where exception was thrown ---
       at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
       at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
       at HedgeModManager.MainWindow.<SaveModsDB>d__42.MoveNext() in D:\a\HedgeModManager\HedgeModManager\HedgeModManager\UI\MainWindow.xaml.cs:line 651
    --- End of stack trace from previous location where exception was thrown ---
       at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
       at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
       at HedgeModManager.MainWindow.<SaveConfig>d__54.MoveNext() in D:\a\HedgeModManager\HedgeModManager\HedgeModManager\UI\MainWindow.xaml.cs:line 982
    InnerException: 
```

**Comment by Sajidur78** [comment]
@sharpemu6898 Could you send a list of codes you have activated?

**Comment by thesupersonic16** [comment]
Appears this issue only occurs when about any code is enabled. I have removed 311647b3e6bc59a7d2bec08ef8d8dfd9de63b272 and 5019caec331c0ef84d4450638020be53e2bfce03 so nobody else will beable to download the faulty releases.

**Comment by Sajidur78** [comment]
The issue has been fixed on 9adfc5e, closing. Feel free to re-open if it persists.
